##Parameter for collision_avoidance.cpp

##  期望位置_x;
target_x : 8

## 期望位置_y
target_y : 0

##追踪部分位置环P
p_xy : 1

## 追踪部分速度限幅
vel_track_max : 1.0
